initialization/src/utils.ts

```diff
@@ -50,35 +50,35 @@ function createDummySecret() {
 	return 'aaa';
 }
 
-/**
- * Checks whether the '/core/secrets' folder does already exist.
- * If not, the path will be created.
- */
+/*
+* Checks whether the '/core/secrets' folder does already exist.
```

Comment:
Comment is outdated

Reply:
You seem to be referencing core/secrets in comments throughout this file. Please fix all occurences :)

Reply:
fixed